client/my-sites/stats/activity-log/index.jsx

```diff
@@ -604,7 +604,7 @@ class ActivityLog extends Component {
 				<StatsFirstView />
 				<SidebarNavigation />
 				<StatsNavigation selectedItem={ 'activity' } siteId={ siteId } slug={ slug } />
-				{ isRewindActive && ! hasMainCredentials && <ActivityLogCredentialsNotice /> }
+				{ 'awaiting_credentials' === rewindState.state && <ActivityLogCredentialsNotice /> }
```

Comment:
this should be `awaitingCredentials`, no?

Reply:
oops, forgot to transform. how about introducing `lodash.camelCase()` in https://github.com/Automattic/wp-calypso/blob/master/client/state/data-layer/wpcom/sites/rewind/api-transformer.js#L29 on `state`

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/20481/commits/f59245b365cd24de61fa7bcf3d2dd74d912bfc5b